src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -712,6 +712,17 @@ HRESULT STDMETHODCALLTYPE CorProfiler::JITCompilationStarted(
       Warn("JITCompilationStarted: Call to RunILStartupHook() failed for ", module_id, " ", function_token);
       return S_OK;
     }
+
+    if (is_desktop_iis) {
+      hr = AddIISPreStartInitFlags(module_id,
```

Comment:
😅 another method to port to ReJIT when I start to work in NGEN images support... hehe